Formula/m/msc-generator.rb

```diff
@@ -56,19 +55,6 @@ class MscGenerator < Formula
     cause "needs std::range"
   end
 
-  # Use upstream MR to fix build on macOS. Remove if merged and in release.
-  # Issue ref: https://gitlab.com/msc-generator/msc-generator/-/issues/90
-  # PR ref: https://gitlab.com/msc-generator/msc-generator/-/merge_requests/154
-  patch do
-    url "https://gitlab.com/msc-generator/msc-generator/-/commit/a4d9ef77f797d33970882b1d7639b872f2ced126.diff"
-    sha256 "0cae8315f0aafdb68ccfe94b15da278cd28355a65e7fe30bfc4475d1d0f117e0"
-  end
-
-  # Backport compatibility with graphviz 9.0
-  # TODO: Remove in next release along with `autoreconf` usage in install
```

Comment:
This comment is for removing following lines:
https://github.com/Homebrew/homebrew-core/blob/ffd36fa1328b521eea915893b429a26ef730dc40/Formula/m/msc-generator.rb#L89-L91

Reply:
good call out

Reply:
cc @Porkepix

Reply:
Those steps are for building from git. We use the official tarball. For autotools-based formula, we prefer to use existing configure script and only regenerate if there is something wrong (e.g. too old). If we ever decide to support HEAD, then we would need to run `autogen`.